upgrade-prechecks/ocp_checks.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# Inherit the shell variables in the subprocesses
+# Useful for the -v flag
+export SHELLOPTS
+
+# https://betterdev.blog/minimal-safe-bash-script-template/
+
+#set -Eeuo pipefail
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+IFS=$'\n\t'
+
+# shellcheck disable=2164
+cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1
+
+# shellcheck disable=SC1091
+source $(pwd)/utils
+
+#trap cleanup SIGINT SIGTERM ERR EXIT
+export ERRORFILE=$(mktemp)
+trap "rm ${ERRORFILE}" EXIT
+
+errors=0
+# Flags
+INFO=1
+CHECKS=1
+PRE=0
+SSH=1
+LIST=0
+SINGLE=0
+RESULTS_ONLY=0
+SCRIPT_PROVIDED=''
+RESTART_THRESHOLD=${RESTART_THRESHOLD:=10} #arbitray
+THRASHING_THRESHOLD=${THRASHING_THRESHOLD:=10}
+
+OCDEBUGIMAGE=${OCDEBUGIMAGE:=registry.redhat.io/rhel8/support-tools:latest}
+OSETOOLSIMAGE=${OSETOOLSIMAGE:=registry.redhat.io/openshift4/ose-tools-rhel8:latest}
```

Comment:
remove this and rerun script so we'll know if it is being used anywhere and if found, remove that arguments